azure-iot-device/tests/iothub/test_sync_clients.py

```diff
@@ -2417,6 +2570,76 @@ class TestIoTHubModuleClientReceiveTwinDesiredPropertiesPatch(
     pass
 
 
+@pytest.mark.describe("IoTHubModuleClient (Synchronous) -.invoke_method()")
```

Comment:
Here and everywhere in this file - please add a space between the dash and the unit under test